tst/manual.tst

```diff
@@ -36,9 +36,11 @@ false
 #
 gap> g9 := ReducibleNilpotentMatGroup(2,36,5,2);
 <matrix group with 21 generators>
-gap> SylowSubgroupsOfNilpotentFFMatGroup(g9);
-[ <matrix group with 5 generators>, <matrix group with 6 generators>, 
-  <matrix group with 1 generators> ]
```

Comment:
Strictly speaking this test/example is taken from the manual (hence the name `manual.tst`). But I don't see a major issue with that *shrug*